lib/support/live_select_web/live/showcase_live.ex

```diff
@@ -67,9 +67,15 @@ defmodule LiveSelectWeb.ShowcaseLive do
       field(:allow_clear, :boolean)
       field(:debounce, :integer, default: Component.default_opts()[:debounce])
       field(:disabled, :boolean)
+      field(:custom_option_html, :boolean)
```

Comment:
I love that you went out of the way to create this toggle to show how to style the options as checkboxes!

Can we please rename it to something more specific like "Options styles as checkboxes" ?